src/writeData/utils/parsePluginsConfigurationText.ts

```diff
@@ -260,12 +260,26 @@ const fetch = path => {
 }
 
 const formatConfigurationText = configurationText => {
-  return (
-    configurationText
-      .split('\n')
-      .map(text => text.slice(1))
-      .join('\n') + '\n'
-  )
+  const configurationLines = configurationText.split('\n')
+  const isCommented = configurationLines.every(line => line[0] === '#')
+
+  if (isCommented) {
+    return (
+      configurationLines
+        .map(text => {
+          let sliceCounter = 0
+          if (text[0] === '#') {
+            sliceCounter += 1
+          }
+          if (text[1] === ' ') {
+            sliceCounter += 1
+          }
```

Comment:
this will work for any comment like:

```
#comment
# comment
```

yes ? will it work on

```
#       comment
##   comment
#     # comment
```
? a test might be helpful here

Reply:
This part only happens when we know for sure the entire block starts is commented (every line starts with '#') like this
```
# # comment
# [[input.someplugin]]
#   importantArg=value
#   # some other comment about args
#  anotherArg=otherValue
#
```

So, we check if the first character is the comment. This is mostly unnecessary but just to be safe. Then we check if the next value is a space. It seems like commented blocks in `telegraf.conf` will always have a space following the leading comment. We want to trim that off. But only a single space, because we want to preserve the indentation.

Comments inside a commented block should follow the pattern. If they happen to have additional spacing before their own "#" then it really doesn't matter for purposes of the configuration file being valid. But I really don't think this would happen, as there is already additional pattern matching before we even get to this point of formatting the actual text.

Reply:
I just found one edge case: `zipkin.conf`

But the exception here was for what is considered a commented block rather than weird spacing on the comments.

I pushed up a fix to account for cases like `zipkin.conf`